apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.YCSB/Google.Cloud.Spanner.Data.YCSB.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <StartupObject>Google.Cloud.Spanner.Data.YCSB.Program</StartupObject>
```

Comment:
Nit: I'd make this Ycsb rather than YCSB (like Http etc). Not terribly important, I understand.